src/timeline/vis-styles.css

```diff
@@ -0,0 +1,16 @@
+.vis-item {
+    background-color: #cab08f;
```

Comment:
if you use scss you can create name colors and then use them throughout your css. You could have, like "accent color" and then if you ever want that accent color to change you only have to change it in one spot.